.github/workflows/check_latest_release.yml

```diff
@@ -0,0 +1,48 @@
+name: check_latest_release
+
+on:
+  release:
+    types: [released]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      release_tag:
+        description: The GitHub release tag corresponding to the pypi releases to test
```

Comment:
let's capitalize PyPI (and other things) correctly in places were it's ok to mix caps.

Reply:
```suggestion
        description: The GitHub release tag corresponding to the PyPI releases to test
```